appveyor.yml

```diff
@@ -0,0 +1,16 @@
+clone_folder: C:\projects\neon-lang\
```

Comment:
Ok, that is fair enough... I forgot that Appveyor defaults to c:\projects\<project>... But, that doesn't change the SCons calls, right?  I mean, it might pull the files INTO c:\projects\neon-lang, but that does NOT mean that it makes it the current directory...  Hence, the need for the --directory parameter, *OR* the CD c:\projects\neon-lang...  Which is why I wanted the paths "hard-coded" in the *BUILD SYSTEM* so that we were ALWAYS sure where it went to...